Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs

```diff
@@ -155,6 +184,46 @@ public QuestionRepository(TrappistDbContext dbContext)
             }
         }
 
+        public async Task<QuestionAC> GetQuestionByIdAsync(int id)
+        {
+            var questionAC = new QuestionAC();
+
+            var question = await _dbContext.Question
```

Comment:
Please use find if you are fetching data based on id, then do the explicit loading.